### PR TITLE
Create scurve

### PR DESCRIPTION
**Pull request by abdnaqeeb**
not connected to the backend yet

### CODE REVIEW

**Review by jshortz — CHANGES_REQUESTED** [review]
Why is so much of App.js rewritten (agree with changing all single quotes to double for consistency, but it looks like a lot of the Routers were also overwritten without changes - this can cause issues)?

s-curve should be in the logged in Navbar (is currently in the logged out navbar), I think?

Suggest adding margins to the div holding the chart so that the graph doesn't fill the whole screen.

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,57 +1,56 @@
...
+import MyProjects from "./components/Projects/MyProject";
+import WhyPerflow from "./components/WhyPerflow/WhyPerflow";
 
-import ProjPanel from './components/Projects/ProjPanel';
-import EditProject from './components/Projects/EditProject';
+import ProjPanel from "./components/Projects/ProjPanel";
+import EditProject from "./components/Projects/EditProject";
 
-import Navbar from './components/Navbar';
+import Navbar from "./components/Navbar";
```
Comment (MelanyCristy) [inline frontend/src/App.js:11]: The recommendable is use a single quote. So, it was fine before but it's okay

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,57 +1,56 @@
...
 
-import ProjectForm from './components/Projects/ProjectForm';
-import ProjectsPage from './components/Projects/ProjectsPage';
+import ProjectForm from "./components/Projects/ProjectForm";
+import ProjectsPage from "./components/Projects/ProjectsPage";
```
Comment (MelanyCristy) [inline frontend/src/App.js:15]: If you were reformatting, you missed a few blank spaces

File: frontend/src/App.js [hunk 3]
```diff
@@ -1,57 +1,56 @@
...
 
 import LoginPage from "./components/Login/LoginPage";
 import RegistrationPage from "./components/Login/RegistrationPage";
-import Auth from "./services/Auth"
+import Auth from "./services/Auth";
 import LoggedInNavbar from "./components/layout/LoggedInNavbar";
```
Comment (MelanyCristy) [inline frontend/src/App.js:20]: blank space

File: package-lock.json [hunk 4]
```diff
@@ -1,173 +1,3 @@
 {
-  "requires": true,
```
Comment (MelanyCristy) [inline package-lock.json:2]: Why is all this deleted from the package-lock.json?

**Review by MelanyCristy — COMMENTED** [review]
Check the comments i made in your pull request, reply them and/or make the changes suggest it.

File: frontend/src/App.js (hunk 1 above)
Comment (MelanyCristy) [reply frontend/src/App.js:11]: Also you only changed it in one component (App.js) so this creates inconsistency in our code. I suggest to put it back as it was before or someone eventually will do it in a different task.

File: frontend/src/components/Navbar.js [hunk 5]
```diff
@@ -23,6 +23,9 @@ class Navbar extends React.Component {
                             <li class="nav-item">
                                 <a class="nav-link" href="/LoginPage">Login</a>
                             </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="/scurve">Scurve</a>
```
Comment (MelanyCristy) [inline frontend/src/components/Navbar.js:27]: Why have the link in the navbar? Isn’t supposed to be inside the Project Page?

**Comment by abdnaqeeb** [comment]
It is overridden by link-projectpanel-to-scurve pull request

**Pull request closed without merging**